scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -32,13 +32,14 @@ def get_number_of_partitions(dataset_name, project_id):
     return math.ceil(max_table_num[0])
 
 
-def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix):
+def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix, new_sample_cutoff=None):
     num_of_tables = get_number_of_partitions(dataset_name, project_id)
 
     for i in range(1, num_of_tables + 1):
```

Comment:
The starting index could be set to the appropriate value for `new_sample_cutoff` if that is defined.
```suggestion
    def superpartition_for(sample_id):
        if sample_id % 4000 == 0:
            return sample_id // 4000
        else:
            return sample_id // 4000 + 1  
    
    start_table = 1 if not new_sample_cutoff else superpartition_for(new_sample_cutoff)
    for i in range(start_table, num_of_tables + 1):
```